lib/build.js

```diff
@@ -366,7 +373,7 @@ const buildAct = config => ({act, oog, pass, name, hasGas, gas, gas_raw}) => {
       "k": "#execute ~> CONTINUATION => " + ((act.internal && pass && !(oog)) ? "#execute ~> CONTINUATION" : "#halt ~> CONTINUATION"),
     "ethereum.evm.callState.programBytes": `#parseByteStack("0x${config.contracts[act.subject].bin_runtime}")`,
     "ethereum.evm.callState.program":  `#asMapOpCodes(#dasmOpCodes(#parseByteStack("0x${config.contracts[act.subject].bin_runtime}"), PETERSBURG))`,
-    "ethereum.evm.callState.callData": act.callData + ` => _`,
+    "ethereum.evm.callState.callData": (act.callData || '_') + ` => _`,
```

Comment:
act.callData is already set to `_` for internal specs

Reply:
the new fined "lemma" thing is not internal and somehow it was undefined here so i had to put that here

Reply:
Really? This line suggests otherwise https://github.com/dapphub/klab/pull/182/files#diff-26b681bd4dac10f25d5293c7595c435bR159

Reply:
But it misses crucial stuff like an interface where the calldata is build from, it doesn't have all the information an internal function has